packages/siri-sx-generator/index.ts

```diff
@@ -1,16 +1,45 @@
-import { GetObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import { GetObjectCommand, S3Client, PutObjectCommand } from "@aws-sdk/client-s3";
 import { S3Event } from "aws-lambda";
+import { toXML } from "jstoxml";
 
 const s3Client = new S3Client({ region: "eu-west-2" });
 
+const uploadToS3 = async (data: string, filename: string) => {
+    // Unique bucket name
+    const bucketName = process.env.SIRI_SX_BUCKET_NAME || "";
```

Comment:
i wonder if it would be better to cast this to a string instead of defaulting to "" - just because we know it will break either way if its not in the env variables.

Reply:
refactored this anyways